api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1079,16 +1182,16 @@ type ManagedEtcdStorageSpec struct {
 // PersistentVolumeEtcdStorageSpec is the configuration for PersistentVolume
 // etcd storage.
 type PersistentVolumeEtcdStorageSpec struct {
-	// StorageClassName is the StorageClass of the data volume for each etcd member.
-	//
+	// storageClassName is the StorageClass of the data volume for each etcd member.
 	// See https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1.
-	//
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf", message="storageClassName is immutable"
 	// +optional
 	// +immutable
 	StorageClassName *string `json:"storageClassName,omitempty"`
 
-	// Size is the minimum size of the data volume for each etcd member.
-	//
+	// size is the minimum size of the data volume for each etcd member.
+	// Default is 8Gi.
```

Comment:
Is there a minimum size?

You can use the CEL quantity library to validate min/max/valid quantities on this field